src/openrct2/park/ParkFile.cpp

```diff
@@ -480,13 +481,14 @@ namespace OpenRCT2
                     const uint8_t isPaused = (gGamePaused & GAME_PAUSED_NORMAL);
                     cs.Write(isPaused);
                 }
-                cs.ReadWrite(gCurrentTicks);
+                cs.ReadWrite(gameState.CurrentTicks);
                 if (cs.GetMode() == OrcaStream::Mode::READING)
                 {
                     uint16_t monthTicks;
                     uint32_t monthsElapsed;
                     cs.ReadWrite(monthTicks);
                     cs.ReadWrite(monthsElapsed);
+                    // TODO: Use the passed gameState instead of the global one.
```

Comment:
Any reason not to do this right away?

Reply:
I actually forgot to edit it, good catch.